crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -805,27 +798,34 @@ export.input_note_get_recipient
     # => [RECIPIENT, pad(12)]
 end
```

Comment:
Should we also update `input_note_get_recipient` to follow the same methodology as the rest of input note procedures?